### DISCUSSION TITLE
Stakes in solana I have my withdraw keys but there is no

### DISCUSSION THREAD

**Discussion by Tinabates** [community/community#142989] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

My stakes on solana have no  web hook or endpoints. Since December 25 2023 even after the audits I have no extension token program extension and no endpoints, my stakes in no way connect to my wallet unless I go to stakes I’d never know. When I go there my keys appear to withdraw but when I do it goes to the trading platform and I don’t trade at all amm cex or DEX. I’d like to reverse this go to DEX and cex I use to have a gambling issue. So this I’m not happy with at all. How can I send to my wallet instead and how do I change my other stakes to do the same. Also I’m asking how do I connect my wallet to stakes. Helios brought me to solan from trim and just left me hanging my solan pay I never even received

**Comment by LiteBrite82** [comment]
Hi @Tinabates ,

Thanks for being a part of the GitHub Community, we're glad you're here!

It's possible that another GitHub user might have run into this same issue and can help, but the GitHub Community Discussions focuses primarily on topics related to GitHub itself or collaboration on project development and ideas. We want to make sure you’re getting the best support you can, but this space may not be the right place for this particular topic.

Best of luck!

**Comment by Tinabates** [comment]
Thank you they have no instructions after you withdrawal stakes on how the hell I remove or pick up stakes I want all my stakes out I’m so upset. Well frustrated with all of them Jupiter’s impaticular
